### PR TITLE
feat(company, project, contact) : new templates to gather basic project information

### PR DESCRIPTION
**Pull request by dselman**
Adds three new logic-less templates to gather basic project information.

### Changes
- company-information
- contact-information
- project-information

### Flags
- Waiting on merge of https://github.com/accordproject/models/pull/118 to update the URL for the timezone model

Signed-off-by: Dan Selman <danscode@selman.org>

### CODE REVIEW

File: src/contact-information/model/model.cto [hunk 1]
```diff
@@ -0,0 +1,19 @@
+namespace org.accordproject.contactinformation
+
+import org.accordproject.cicero.contract.* from https://models.accordproject.org/cicero/contract.cto
+import org.accordproject.cicero.runtime.* from https://models.accordproject.org/cicero/runtime.cto
+import org.accordproject.timezone.TimeZone from https://deploy-preview-118--accordproject-models.netlify.app/timezone.cto
```
Comment (jeromesimeon) [inline src/contact-information/model/model.cto:5]: Those should point to the public models.

File: src/company-information/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "license": "Apache-2.0",
+  "accordproject": {
+    "template": "clause",
+    "cicero": "^0.21.0"
+  },
+  "devDependencies": {
+    "cucumber": "^5.1.0"
+  },
+  "scripts": {
+    "parse": "cicero parse",
```
Comment (jeromesimeon) [inline src/company-information/package.json:15]: not needed?

**Review by jeromesimeon — CHANGES_REQUESTED** [review]
- Couple of small things
- No tests?

File: src/contact-information/model/model.cto (hunk 1 above)
Comment (dselman) [reply src/contact-information/model/model.cto:5]: Yes, I will update once the models PR is merged.

File: src/company-information/package.json (hunk 2 above)
Comment (dselman) [reply src/company-information/package.json:15]: I will remove

**Comment by dselman** [comment]
> * No tests?

No logic...

**Review by jeromesimeon — APPROVED** [review]

**Pull request merged**